pkg/apis/jaegertracing/v1/jaeger_types.go

```diff
@@ -177,6 +177,9 @@ type JaegerQuerySpec struct {
 	// +optional
 	Options Options `json:"options,omitempty"`
 
+	// +optional
+	TokenPropagation bool `json:"tokenPropagation,omitempty"`
```

Comment:
Not sure we should add this to the `JaegerQuerySpec` yet - as it is already available via the `Options`. We might want to in the future, but unless there is a specific clear benefit of promoting this option, when we haven't for others, then it is better for now to leave it.

Reply:
Ok right, this is only for testing and won't be available to general use. I'll remove it and use options,

Reply:
+1 on removal for now